### ISSUE TITLE
[2023] [Day 25]: Snowverload

### ISSUE THREAD

**Issue by victoraugustofd** [victoraugustofd/advent-of-code#225]
### Challenge description

Still somehow without snow, you go to the last place you haven't checked: the center of Snow Island, directly below the waterfall.

Here, someone has clearly been trying to fix the problem. Scattered everywhere are hundreds of weather machines, almanacs, communication modules, hoof prints, machine parts, mirrors, lenses, and so on.

Somehow, everything has been wired together into a massive snow-producing apparatus, but nothing seems to be running. You check a tiny screen on one of the communication modules: ```Error 2023```. It doesn't say what ```Error 2023``` means, but it does have the phone number for a support line printed on it.

"Hi, you've reached Weather Machines And So On, Inc. How can I help you?" You explain the situation.

"Error 2023, you say? Why, that's a power overload error, of course! It means you have too many components plugged in. Try unplugging some components and--" You explain that there are hundreds of components here and you're in a bit of a hurry.

"Well, let's see how bad it is; do you see a big red reset button somewhere? It should be on its own module. If you push it, it probably won't fix anything, but it'll report how overloaded things are." After a minute or two, you find the reset button; it's so big that it takes two hands just to get enough leverage to push it. Its screen then displays:

```
SYSTEM OVERLOAD!

Connected components would require
power equal to at least 100 stars!
```

"Wait, how many components did you say are plugged in? With that much equipment, you could produce snow for an entire--" You disconnect the call.

You have nowhere near that many stars - you need to find a way to disconnect at least half of the equipment here, but it's already Christmas! You only have time to disconnect three wires.

Fortunately, someone left a wiring diagram (your puzzle input) that shows how the components are connected. For example:

```
jqt: rhn xhk nvd
rsh: frs pzl lsr
xhk: hfx
cmg: qnr nvd lhk bvb
rhn: xhk bvb hfx
bvb: xhk hfx
pzl: lsr hfx nvd
qnr: nvd
ntq: jqt hfx bvb xhk
nvd: lhk
lsr: lhk
rzs: qnr cmg lsr rsh
frs: qnr lhk lsr
```

Each line shows the name of a component, a colon, and then a list of other components to which that component is connected. Connections aren't directional; ```abc: xyz``` and ```xyz: abc``` both represent the same configuration. Each connection between two components is represented only once, so some components might only ever appear on the left or right side of a colon.

In this example, if you disconnect the wire between ```hfx```/```pzl```, the wire between ```bvb```/```cmg```, and the wire between ```nvd```/```jqt```, you will divide the components into two separate, disconnected groups:

- ```9``` components: ```cmg```, ```frs```, ```lhk```, ```lsr```, ```nvd```, ```pzl```, ```qnr```, ```rsh```, and ```rzs```.
- ```6``` components: ```bvb```, ```hfx```, ```jqt```, ```ntq```, ```rhn```, and ```xhk```.

Multiplying the sizes of these groups together produces ```54```.

Find the three wires you need to disconnect in order to divide the components into two separate groups. What do you get if you multiply the sizes of these two groups together?

### Challenge input

```
hss: xnm
bnj: tbv mnb
fvh: flx nxs
pxd: krj zjl
bcq: kln scz
lbz: tnj ghr lpp vqq
mpp: bgb xbj vcd lmn
krs: pll
ttj: lld bzq
rxm: bvt
spk: rlk
snl: bcm psf rtx fkj
kdm: txr
hjg: pxc tdz lvp vxs
kml: lrk
fvk: hkb tbl stv lfr fpn
gvm: mfq vlz ghs ntq
xcf: qzt ghp psf
mbl: qsf tvq
sfv: fxt ssk
nmv: csv dpx bkx xzl jmm
gfv: vxr
lkq: qsq
gln: dtf jbs
bxq: lvn
zjz: thh rdp zhc fpd
kbx: rjx jcv vbg qgv
pxp: kvk ffz gzm
dqx: rhr xln jxn
qbl: tbv
jgz: mbn nrq csq
qbz: vkc
psz: lzd nzb
qlm: dvf
rdp: qzt
vxd: vdr ghr hcs trh tth rsq
dnh: mjg jfr knm fsb
thn: bgq vnv zlt
gtx: vnv
lpl: fvr vbq
lrk: vcp
rlm: hxq pbm
lfl: lvn ttd
hdc: krk
hgl: vdj blr smk
fdb: pgr jld mcd ffz
kms: sxs zdc pnh pkn
htn: gkk pdm mqt
szg: xtz
ztb: tqj djk fdh dhq
kzb: vnl rjt czm gcm
ddl: rtn hts phr pqb zdv tbq jsj
cdf: vqm lgk dmj
cvq: pcv
dhq: tzx kjt thn
bmc: gpj
mpr: snr mtm
mpd: pnj ghb rzq lzs
zxv: qzm
dnx: dlq
bdb: zgr tfj rtn zxj
xqd: lzd djl npz dmr njl xxl
stv: bnj
ktd: bjg mqz lmf xmg
qks: sbv lvb vkg tvq
hlx: rvf spk rtk
xkq: cvq
dnk: zcn vxc xcl htn
zzq: xxm
dft: pvr
xvz: ppt lsg sml blr sdt
crq: kkx cgr ntn lft rhr krp
hjq: zxs xtp
drd: vdh kxb kxz hgj
bfx: nrk prv
xpb: mds gkk xcl
nrb: nxj xkk
rzt: qfl xqc zcn
pdt: zjl rkz sbh nxm vzj vcp
cxx: mrh rzq mfd vpr
nqt: vxc sxs lsk jzh
mzd: jcm rmx jvd kjr bvg jvl
nsv: nkx cqk jdf
rkc: bhr gbr
hkh: gkt thx ttx jgm
mhf: thd mjs nph bzj
nsl: vhg
drv: lgv cld mhs sjh hlq
qvv: fkr bzj ttv bjg grn
zgv: xvm fsp slz frt
fsp: lvp
dxf: vcv rnt
fsb: lkx
vmm: znc rqj nhn tbm xzf crv
nxx: gsm xvj dqx lsj
ktr: nmz jxx kjt
ckc: bgq lsk cxh
tqj: ptl grl
kqr: qgf zbj
tth: gcm
fll: vtz
ckl: bpm dps dvn
rrz: fzk mjf
snm: zqr dgs rrz
vgm: mrp
xlb: tqs hbx qxs
xpg: jzb lmd gfv fvh
gqz: djp tdl nsj
bcc: nmk hhz
jxx: hkb dvn
xch: ckp bhr phf
rjp: zqq rrz ttx
lbc: jsr cvd
gbr: bqq
ccj: mvc sfm rft hzg xtz lpm
ddd: gqb
svp: qxj xmg
sbp: fdv ftk cdj npz
rtx: pgr lpl pfz vkc tbp
tgc: qsc zsz
xlk: bzd
zml: xdk dps lpq
hps: lhb fxs jzm
smn: tnj hgc kln qgv vbq
cls: ssp lpt ftg rxm
dqg: pnj jsr qlt vzp
zjv: mvd mjs ksb
pjq: fvl vfc
mbn: gjp
mqz: cbv
fcf: hsl
ncj: cxt bcr xcv nvx
bzt: str zhp ftk nzb
qbn: sfn slz fjl czm
bkm: gtg
sqx: jqd cdr phk
bhr: xkk
ktt: pkp
xmb: sft rbg qqz nzb vbg
ndv: hdc dgs
ntv: fsf ffg vdd tqd
rqm: vms sng
sbx: dfv
prg: qcp hzs fmg qjr
tcs: lsk snm pll zlj
kbf: rmg krb dks
sbh: xpk bsx mcq sdj
tcb: hgs nsx fdv
kfn: jbs lrr
zmv: qzm
mfk: cdh dhq lkq
zdq: qsc rld slz
xfd: dqt hxp cjq rmx
dql: cmq
gdv: xxl
tsn: prv
pvh: jzk hmj vqm rvx
dmq: hhz rgl
czm: bxk
zmn: vtj
pfz: mrp nrb ffg
kvr: dcd zvr fcr rhn czd
xxf: gfv phk tjd thb
fbj: psr cqk ncq
fdz: ncr kcx bgn
rrk: lpk
fkk: fdv nvh kzc
bpp: nzx mhd pvr tfx
kxz: hkj gsg rbr
fxt: psr dgs fkp bgq
ltv: pbl zhl zvr xtp
hkg: jbc lgt hfq krb
jrc: str jjq bvt
mvd: vdr gdf
skf: bbc xcl jrs ctr
cvj: bxm qsf
dtv: jfr vkc
sgr: xvj nxl phj
nsh: cbv cxm zsr
dpx: pqd rbr
snf: zvc qgv gzq rzn
lfc: gzs
vqp: zhc
mhc: hnc czm
djb: mhh gcs mfx zdg pxp
pbs: tnc brr gbq
gcd: cdn
cjq: nhv dgb qvg
pxc: fhs qgf xdt
nzt: rqh
mrz: qpc
nrk: fpx vmt
fxs: bzd vnl phx knv
tjq: vgf jbr
jbx: jjq sml
xlh: vqp kdj psx phx
rft: knv fxj
rqj: mtm qbz
sbv: bzd
rzq: ltv dnx
jlm: vhg zlg ktd zbd
hzg: dgb
ljl: lnr fsl nmm xcg
klc: zdq nsl
kmh: rtn ncc ntl nsx
phb: vcs
mvg: xhx
pbq: jfp kzl tmh jzb
gql: fgh ctr vtp vmq
sjh: xbr jbc xnx
vxc: nkv xtp hzk dpn
pfs: zcc
cmh: vtj ndl dnj fkj
ztt: lgq hgj
tbz: mcd jht
tvz: tvv
xsf: lbb bml qlm
vkg: tbm rbr hjb
pkp: sdj
ghr: rhr vst zmn
mgj: hlm tdl qfl zcn ktr
tdz: pkp kxn
grl: qsq glk
fjz: nlt kzl zlt cpn
kpg: sfn fbd
jdg: ftk vtc pnc vzb
rzn: vzr
bvt: hbm
ljt: kfs vcs xkd
zjk: qvs txr dxp
snk: glk rzj czd xfm
qmg: dhz
tqs: sft
kkp: mkd tzp ltd rqh zbb zjg
sxk: jnb lfc fzr
plg: xqc lbb hbz sbx mmr nks tlp
qgs: lmf pzb vbg kml
qfl: hmx
gdr: psz qgf cdn
ftg: nhv cxm
dps: bxq
lfr: xcv ssn
lqp: ctt lzm
lgs: rdp dmr hbm zsr
txf: fvp nph crj ntl
qpx: glg qct dzh ttx
bkb: nph
btl: xjk hpc vbx mfk sjz
rtk: mbn
bpb: xhj tgv jmf szg bvz
qjb: tlp bgr
phr: mtz qsg ztt
qsf: rnt
bck: lvp hcs vmx jjq
zrs: fvc lxq bmc
ddr: pzx hxn gjp vms
dlq: rtb
txr: tbl
zhs: sbc lfc zjk ndk
kfs: qvs xxd
snq: gbr sdn
dpn: qcx ctr
tkq: qpg zqb kln
nvs: jrs hlx lhs knq vmz
jkl: vxr sdt jzk klr
lrz: pjl
fsl: fvp rld thd
xms: bdp rmt
klh: szl kbl fpg xbh
mrn: cfh rmg mtb
pdj: nnf tmd drr
lfb: pjp btk
nfr: phj szl hnc
fbn: jgz qlk dhf qlz zzd lbc
lpt: vfv gcm cbp
jzm: njl zmp
cfh: flx sbm
hlq: xgl jqf jmp krs
rdv: xhd ptl lpq ncj dhx
tlc: jqp nks mdx ngj fzr mfg
rkk: rtk nfl qkv fgd lrz
njm: hmx jzk
plx: gpp lzm drd
grv: kkx nbz hnh rdp
pcz: pfs djl jvc tqs xhg
hrm: dtg
kzn: hss xpn snx
tjt: mhq ddn
cmv: djp qbl
ffl: fxd lrk
gpj: dmq fgd
prj: nmt tjt
rfq: xzf xtq svn hbm rtj
fft: nnf zdc pdj ksv
jkh: mgh mrh rvt fgh rhq
jqd: gkt zhl
spr: nsj mbp
txg: xsz szh sps ttv tvf
zmp: zcc vst
mfp: hnr zlt
dcn: mpr zbb kvk
vkx: pxd zpv qsg rtf xds
qnm: tbz lcj sts rjx xzl
nqn: bvt cdj
krp: cdn bkx
hxp: pmm xkh
qsv: xds
sbf: tjd dmf cms
hvj: rzl gdm bml xlv
grp: xlk drm mfm vhs
znz: dkc nvf
mjf: hdz tbv
gkx: kjr phj plp rqj
xxj: rfc xhd
nzx: fck qsv fxj lkd
gph: cmv lkq xsr ttn
pbg: rzj
vkh: ggb lrr
kxr: kdj npf mtr jvl hgm
klk: scv vtc fnm zxj vmx
qgx: cdg mxp xhg jrq
kcv: pmk khg fdc smz
hlh: nxb vnr vzb pjb
kbt: xrj shs vpf brh fzk
lxt: zpb zkd pvk
vmx: gcm lkx
cqh: snj jfp xlv gks
jqk: slz
lpm: qmg hgm
nqb: vsb bsp dvf jxm
xhd: ttd smk
sfn: szr
xkd: ckd vcj
dmr: tvv
jfc: ztx jsh rsh
qhz: vcs
jfk: tzp gsg fdv grn
dhh: tpj
vmq: xsf dvx xhz hdc
mfd: vpr mkh
cjr: dcv kmx gkj lvz
klm: pdt zqb tth zbc
lsv: nvx prt hss
lpq: zxx
csq: qpc krk
jrs: qfr
jld: hnb rpc
xgs: dmf rrp bfj
pns: djp zml jkl qrf
xrz: zpm xcv qph
bml: qtj
jbt: kxv rth mfl xll
dpf: mtl xxk ktd pbk mxl
xxl: thd
nvd: pfv mjs
zzd: zsj zpm
hnb: rhr
kbq: nfv ggd vlc xtp
jvl: mlq
zrx: hgm
kmp: qsg npf bkd ttz
gcf: pgr sbv pjb nzt
qxj: vzr
zbd: tfr
zlx: tfr rtj gsm vzj kjr
cxm: jbx
pkn: clv lfl pnj
gkt: rhq
jtz: pht qjl dkd
qrc: fzj
knv: cjj tvq
tfj: snq nxb lng ttb
cxp: fzm lmm tqh fdz ngj thz
lfn: zlt sqx dfv
pfv: pgk ksz
zrv: svn nhv xvj
dfp: tqh gqz ggb
fvl: prv qln
lzh: grp nmm vhs lxt dtg
jtr: nrt vkg gxm
krb: mgh bsp
ptl: rtb
phx: vst
zqr: qlz skk scg
jgm: pjg fhl bcr
ndz: xds lrk jrc
mtk: mds pdr vmz
hgq: sjb fvc fqd nmk ggg
dkf: cdb prx tjd
brx: nrq lsl fvh krs
qsd: tfr tcf
pbk: tqs htx
fsf: vcp crv bkb pdd tbp rpc
hfr: spg xbr tlp fzk dfv
jrf: gft hmj vqn
vrg: jpd tdz
drr: vfg pvf
cvn: str zmv
gzf: zrx
skr: xdk jrs jqd hgl
cnz: tdm cdb krk nvg
zmj: ksv rmd pmk pbl
hcc: bcr bvh klr jxg
jmp: lfb fzf
bsd: xrf cvq rzj fmm
dhg: nkx lzs mqt qrc
zxs: hnn cmq
fbt: hnn nrd
mbp: rzt fmh drr
gxm: ttb tvz bjg
tdr: nxl vcv jjq
dcj: pmk btk
ttv: thq sdn
shg: sdn rzn dkg lpk
rjd: tgq tpj jqk fsb
ggc: ljt tqj lgt dpl
khm: cvx dvj jfc qlm
lkd: qzv xln
vnr: snr dft ccg pjb
ggg: mpx nnz
fdv: vsg
dbn: gft zrs glg pfm xms
mtm: vcp
hxd: lgj jnb xss
sgq: rbg zsr ftk
nkx: fzj
hpc: qrd gkk
gmg: pll xnm prx
bmz: dzh xss kfs
lrq: qmg bcq crf dpg
cmj: phb qln nrd blr pcv
nhr: jfg kcx xbr
plb: rvf mdx zkz ddj mkx
gvd: vsm
fpd: klf
zsz: jcb
spg: xll
dqm: jbr mrz
mpn: zrx gtg phj
gmp: qbz
lcj: rkz ktt sdj
vmg: nbm dks mts
qlt: mrz tqh thb
vxm: rkz rsq tgq xlh xhj
jrq: cxd
jbr: ngj
ljj: qhj gxs jqd cmb
qfr: tsx
dvm: htk qtj dbq
hjt: bkm qzm
ftr: vng sgq zlx cgr
vbs: qfr dps dmj
szl: rfz
vqj: lhj szh jxg bmz
mcq: mrp hts
nxb: bmd fpz
vpv: lxv mhc pxp btj
ktv: ppb sxk ttj lvz vbx
ghp: dhh
znc: hgc tfx kml
cpj: ttz cgr rld
gzp: jqp nmk pbl
qkp: zpm
vkf: hjb vcd jcb bgb
xsr: dcd
mpv: gvf pbs vsg
hmj: nvx xzn
bds: rqv zcc xdt
mxf: vxr fhl sph xgs
hzf: tlp stv
snx: sng
pnh: tbl ndk pbg
cnr: znz jvc rft scz pmm sts
rgb: lqp tnc rpc hnh bcm
tnc: zcc zrx
kcx: sbm dlq rrp
ntl: nxj lgq
tdm: bzq hzf ddm qzh
tqd: qpg sft
cmb: xrh lfl hlg bgf
pdd: slv cxd
mrt: bsx zzq lzm
xrg: nxj qnq
hlc: fxh xll bsp gkt qcx
csv: xpq vqp gmp nhn
mnb: fzf
rvz: kfj cms bjx nkx
pks: zsr pgr nvh qsv
pcv: gzs
tbx: npz cvj mrt ksb qsd dqx
gdm: dnm rvf
vmz: rdz
gcs: fpz jgp hgj
nxt: rlk ndk qbl dhf
xxd: xpn
vbr: nqh rmg dbq
kst: hbx xqx vcd
jnk: czk mrh fll rtb mdx mtb
jhf: vzr
nns: zhl hqg
sln: qjs mhz dqm lxq rtb
ckh: prx hpt fpn
gmh: mvd dkc ltd rhj fvr qfm
xzh: bdp jng nnz rhn vlc
rsq: ttb cjk
cjk: hts pvr
zjh: dpl
qxs: slz tfx xts
vvj: sds bmh ggg mpx
pdc: kjr qqz fvr gdf
fmg: jnh tjt sbc
dvf: pll
bsk: lvt sbx lrt hlg
drm: ccj gbq
lgj: nvg rth gkk ndk
xhg: mhc gzq
knq: zzd
rzx: pbl sml
mfb: xvj sps kdj srg
zfm: vgf xnm fmh
tsx: bgn jxg
kcl: smk nkx
qlk: qtj qss qhz
trt: tvv fpd hlh pqb
dqh: cxr dpx nvn klz
bcl: vpz xrj spg ssn
qrh: nxm ghp nsx
snv: lgq thq tvz
kjc: gzf dxf zmv gbq
tgf: pdm bcr zlj tsf
srg: rqv phj
lvb: hjb fbd
pjn: xxj qnx stv gpj vsm
vmt: fzr xsr
fnb: phk xhm mhz bcc
rmx: qnq ccg
ksv: dkf
thd: jrq
fvd: cbp htx dqf ddl
pzb: phx mcq
ddn: xhz vtn
jks: bxq ddm lvq kdm fvc rgl
xpq: ktd mbl pbk
vzq: znc vrg slv
csh: tbs klf pkp
ssk: kzn xss
gft: ddj jfg
hpt: qnv qph
rmd: bgr lld hnk zdd
ppc: cxr tzp rkc
skh: nnn zjv dcn drk ktt flc mjs
mhq: flx
mgh: fzj
nsj: nmz
sph: zfm rhq phq
qqz: cvj snv hrm hps
mhk: bkx zpv rgc
hcd: zzq jcb
xmg: kxn
mdx: gjp
gzq: vxs cjg
gvf: psx ppc szg cdn
dpg: crv lmn
nfl: hlm xll
dnp: dmj sjb mkx
xhq: fkj dft pdx zkd
cxt: dvf ckd
sxf: nbm xfm
sgt: mts mqt cmq rkl
gqb: qpg tbm
szr: xhf rfz
dqz: lsv xzn vmt gmg
pvf: dtf qvs nsj
prl: qck fgd mds
kzc: pkp cbv
mzc: tpj fpd lpt
mvm: hhz qss
rjt: mkd hzg rrk
dzg: gzs zkz fhv
jzn: fjf
bfj: cmq lsg gvd
fkp: phb qkp
htl: slv
jkg: kvk lkx ntq lnr
xrh: bmc
klf: lzd hgm
zdc: lkk dpl mnc
glj: tsx sbm
rtj: tth prd
sts: ftg
gxs: fhl zzd zdd
qdd: phb
zzt: lpm ghs dgb phj
fkj: lzf
fvp: vzb qld ngh
xgl: sds
xlv: xrj fpx
cdh: mnh hpz kdn zjh
stn: qhz dvx ttx xgl xkq
rlk: glk
dzx: mhs cqk ssn dgs
nhn: fpz nfd
jcm: cdz plp trh dqf
ttx: dpn gvd
xtr: pnc tmg vzn mtt
fdh: qct vqn mvg
fxd: dhh
cgq: qct mqt zpm
dcv: bgf rtb
thb: dql
xhj: fxj kqr
zms: dqt rgc hgc vrg mxt zgz
mrr: sqx qkp fjf
vsb: lvt kjt mpx
nfv: xpb ttd dcv
mhs: hlm qfq
sdh: tvz fkk xqx zpb vcv rld
zql: hcd lrv pvr mhc
rcr: ssn dxp nnf
lhs: jsh
bvz: nrt
zkv: jbc qjb cvx lvn xkq
jpd: zbj
tbs: jgp mdf htl lzf
zqj: fdb zbd ppc tqd lnr
ktc: sdt lld
dhz: lgq hnc
hdd: jxg
mtt: tpj ttz nvn
llf: xrg fnx
kzv: gkk
psx: sfm krj ksz
mtz: xcg
slz: qld
hbz: cvd cms dks mnb
ksb: str ddd
bxm: mpr nbc mcd ndb
ngl: cdn ndb vsg
lnr: slv
nxs: hnn tmh
sbt: qck vmz phb lkk
nlt: ggd prx mfg
kdp: skk
nkv: tmd
lzx: mfp xfm
sbc: pnj hlm
lft: xln jht rbk
pqb: xtz scv
xsz: kzb jcb vkz
nnz: zvr glj
qfm: rxm zmv
pmm: jrq
qhv: hrm bbv llf
tnk: scg jqq zml tkj
mfg: zxx
dnn: kqh zpv gpp tbq cbp
dqt: cxr ngl
fkh: vsb xhx mrr ssk xhm
nsx: vbg
zjn: dfv rhn qfr
xrf: nns nxs clv
nvf: qnq
gvc: nvf fpp zkd qsv
nrr: jmz tkq zxj lrg ssv
cxr: rbg
trv: kbl bvz
kzl: pjq ncr
ngf: lrt rqq mrz bff
cqv: thz lzx cvx
rkz: dft
tgm: fgh
zgz: mqz vcd vqp bqq
qbd: ndm sgr lzf tkq
bzj: vdh qsv ghz bbv
bjx: fzm dmq zjh
ppb: dvj rqq fpn fqd
rkl: lgk dfv
nvg: qdd
gsm: mtz
flc: xds zhp qhf
pjg: kdm dvj rqm fbt
jfb: kqr xvm
krj: rrk rbr zmn
zsr: zjl
jnb: hpd
fjq: hmq hzf xpn
zlg: rzn bsx
rtn: qjl
knm: bjg bkm
crf: knm zpb bkb
jqj: zhp vfv xpk lng
ndx: ztt jtz mzc jtr mdh
mbv: dkd hjt dnj brr hcd
pxm: ggb mkx bmz prv
vpt: mjd cbp cpj bgb gsm
fqd: qrc
vfg: jnb dmq
crj: jxv ffg rbk
bgf: sds
fbd: pkp
sgp: jqk mtm lvp rnv
vsp: ctt gzm xnl zbb
tjd: rhq
qqv: rnx fzf qfq rcr
fcr: vtz
sjd: mfb ttb pgr
cqk: njm zzd
bgb: pdd
vlc: gln
ndm: fbd mfm klz
jgp: bgb gdf hjb
xhf: qnq
qpc: phk
pht: pmm ntr drd tbq gqb
kfj: qpc
nnn: nxm qsc zbd
fpg: ktz gzf pmm
gkq: spk bfx plf
lhj: qlz lbc xpn
kdl: dgb rnt
vgc: bxk sjd hgm vzq
qll: bcq fck cdj
vpf: ktc tdq sxs
jlr: gkh knk hnn cdh
sls: kjr hjb
fxj: vxs nxl
trs: cgf jbt vgn dvx
ncc: mbl hnb
dtc: scv vmx ktz lrg
ndb: qxj
zdv: bvt
gbs: xxj rcf lpq prj
zlj: zpm
jzf: nzt lvp cdn xds
rzd: ftk fsp sgp dqf qfm
bbc: hhz knq kxv
pjp: jqp
vdr: zqb rnt xtz
ssv: bhr hkj
lsf: mhq mnc
xnx: ddj ttn xcj
qrd: sds mqj vqh
thx: ghf fbj hxd rmt tsf vmg
mdf: qld mck
fpl: rgl krb xgl
vfc: fjf rhn skk
lrg: vcv
mhz: htk jzn qtj
nbg: jhf bvg kdl ngl jqk
ckd: mvg vxr
qhf: ndb bds vlh
jfg: bff
gkj: bcr qfq zhl
qjl: hts
dfd: dtf rzj
rhl: kvk zgv mqz tmg hgs
skx: rxb lxq bdp pbl ncq hkb
fpx: dvn qfq
kcz: nxx nvf nvh lsj
bnp: xhf tpj fhs
tvp: gdr klc mtm hnb vhg rnv
zhc: nld xkh
mxm: kqh kzc gmp njl
rcf: btk mfl
nbc: hnh
xgb: dvm ckh mnb pvf
kkk: nqn hbx nld djv
zlf: qpg mtr pfs thh
ztx: qdd rlk
zjl: qxj
xjk: fzk lgk
ckp: vnr gsg tgq
mcp: cdr ngj ndv
jxm: bcr sbx
qrk: nqn zhb knl kbl
lvq: rzx ghf nrk
dxl: slh vxd ktd dkc jfb
bkd: ghp brr
qnv: hxq
plf: dfp txr rsh
zdf: rjr dlq sng lmd jmp rsh
mjs: pjb jbx
xhz: qfl
ndl: ccj rxm vzj
nmm: jzm hnp ntp
hxj: cmq lsk
drk: nrm bbv
mtr: svn
jsq: vcq tbv xxd xhx
dhx: bmc hxq jzk
mdh: vgm rrk
plp: ffg
mjg: hnh svn
sbm: ctr
tkg: dnm lrz dbq
glg: snj kxv
qss: pnj jqp
qlz: qhz bgq
nbr: cjg kll tvz lvb fcf
mhd: qll fdb mpn
tdq: mqt pbg
fqp: rtf nld pvk
ndk: lgk
vdj: mhq fjf kfn
xkk: brr zpv
mxt: rfz
lmd: dfv
psb: kxb gdv zdq lzm
gkl: xch fsp tcf xdt zdv ntp zpb
jdf: ndv zjh hxq kbf
qpj: rfz
kmx: fbt lgv gdm lxq
vzp: nhr vxr dlq
frn: cks hpd rsh mmr
jzh: kfs sxf jqq cqj
hsl: nld rgc xkh gdv
gzm: szr hnc
sgz: jrs fgd qfq sjb vgf
scz: vhg prd
hqg: lvn
bhq: hfq jqs
sfm: vxs
pdr: zmj gpj tmh
cxh: ddm
rqq: xcv gfv
jqr: vcj pvf gdm bcc
kxt: dql vkh dvm rtk
xlx: fvl lnn qlk lrr rqm
nqh: hxj grl qfr
fhq: rjp bgq qcx ckc cdf
tgv: zmv bqq
cpn: zqq hqg prl vnv
hcs: mpv qhv
dvx: nvx
qph: fmh zlt rrp
xxm: gmp jsj
jvd: nxx ttb
fsd: kdp bnj glj
dhf: tbv mrz sml clv
qjr: vqm xsr zxx sbm
brh: tgm thz
hhk: sls thq vkc hlj
cjp: jxm lfn dcj mrz lmd
pbm: lsg ppt
gkh: mnc htn ppt fzr
vvn: lkx fnx
sqp: bnj glk qdd bdp
clj: xrh nxs mvm fpn
crn: ttn mpx sqp hqg
qjs: dvn nrq czk
mrj: cvd hpc tzx jkh
btk: hpd
gcg: zjn hzs qnv hpt
llk: hmq zcn csq dvm
vfv: hkj
bvh: zlj sbt kdm
qzm: sdj
xct: rzj lld vbx jsh
rmt: zxs bgn
jnh: bgn
btr: ntl xxm kbl ctt zbb
jrm: dvn ttd jsm fmm
dbz: zhf tdq lnp xcj
hdl: jmm sgr mcd dpg
bvb: fzj hmx pbg mtb
rvx: ttj gkq dfd
hbm: zqb
jqc: gph lnp xhx dcj vcs
vrn: vdh vfv tbz kkx nfr
lmf: fxd mtz
ssp: drk szg fvr
mts: vgf bff
lmm: hjq pjl krk
ddt: tdr jfr ghz zdv mhd
cdz: nsh bkm vlz
rbk: qsf
xzq: tbv qlm dnx czd ktr
bqq: tcf
ktz: cdg xmg
tbp: xzf str
rfc: dzg mkh dvm
cvx: bzq
zcn: tbl tgm
tvm: sqp fmh xrz vdj
ntn: jpd nkf xpk gtg jmz
xgx: rzl tkg nkt spk spr
lpk: cxd mxt
ghs: ntq ghp
mxp: nzb zxv htl
nvm: qck rrp jzb
nmk: ptl
ntq: cdg vhs
cjj: ccg hkj
shs: mvm ckl vkh
zvc: ctt rbr jxv
zhf: gvd lbn mrh
pvk: slh plx
lzs: lrt pmk
jcb: xxl
zjm: bsd vnf lbn gks ksv vcj
lsj: vst
lzf: ntp
pfh: kzv lrz pzl nrq ghf
cmq: zsj fcr
qnx: hdc ktc vgn dnx
mtl: vhg mfm zck lnr
rxf: kcl cgf zlt hmx hbz
scv: hgs cjg jfb qgf
pqc: mqt phk vcs
knk: hdc qvs flx sjz
pzl: nvm fvl lvt
bgr: scg
zpz: hvq sft zrv fnx
zgr: xlk lpm qsv
skv: pqc rdz jsm ksj
fmm: xss
nrq: dtf rvt
hzk: ckd hfq rzx ggd lvz
zff: thb hlc lld cqv
tbc: rlm lfr xhx
dtg: xcg
tvf: flc tbm cgr
prt: kdp dvn
psr: jxx dmj
bpd: kzv ztx
kll: vbq gdf
rqh: bmd
tbq: cjj
zhb: gcd vxr vzj
lnn: mkx vmz sxf
bbq: nrm xnl tvv vxs
rjr: gzp dks ksv
ncr: qrc
mfq: svp thd pfs
phq: lrt xzh pnb
lxv: gbr sfm lpp
vcx: hdd xpn lkq dhf
smj: lkx szl htl dhz
jxn: nxj kxn
mhx: tsn ktn nrd mfg
tsf: vtn qrc
jrz: nfd znz rnv fxd
xbj: xnl tfx
kxb: lkx
nbz: bmd nrb cdg
bhd: xzf xtz nsl hnc
vgn: gdm qjb
cdj: vnl hnp
nxl: zxj
nrd: mkh
nvh: gdv fnx
pfm: zxx pjp czd
nks: pjp ddn
pnb: kfn gqm
vpz: gtx nkv dqm
ncq: jfg nmk
jfp: lpq bzq
jxv: bnp mlq
htf: ntr hxp tgc npz
pqd: nsl
bzd: rnt
blj: trv mcd jvd sts ndm kxn
lkk: kdp lpq phk
rmg: qss gzs
thz: pnj
ktn: vlc tmd vtz xhz
kzr: xfm rcr lgk gqz
qzh: rxb
jjn: dzh zlj vfg fdc
vlh: crv nrt pgk
sjb: sxf
hnp: gbq
kmq: fgh nmz qln fbn
czk: dhf
nmt: czk
zdp: qnx fhq krk dvx
nvn: rbk vtc
jmv: qgv sps fcf zck
mdl: ghz kll mpn phr
mvc: qsc tfx ksz
rxb: lfc mgh
jmf: mbl tpj psz
lrv: bbv lqp qpj
jzk: dmq
smz: nbm qnv fll
dgs: tsn
mfl: hpd cqj lfb tmd
vpn: hjt vzb ffg
klr: gtx cms
xqx: vbq rkc
kqh: bcm lrg
htx: rbg fsb
zbj: qld
vzn: xpk mrp trh
qhj: vsm rmt
xbh: lgq gdf
qgf: lhb
vbx: fzm
fsk: bpd jnh lkq ghf
dzh: qkp mfd
jcv: xkk rfz bbv
vqn: vpr ppt lkq
hnr: kfn hjq pjl
hgj: fhs mxt dtv
lng: gcd phr hgm
vqq: zdq bvg tmg
jjk: svp pqd kjr ljl
frt: zhc kxb
hbk: hlx qzh lrr
gpp: mhc zdv qbz
qcx: bzq
dgn: jzn djp lvt cxh
hvq: sps rzn
fpr: gln bhq sjz xrh gfv mtb
xkh: xln bkb
rth: cdr rvt
mjd: bzd ntq fqp
ddj: xzn
hxr: djk cxh ghb cdr
vtp: hkb rvf hhz
tfr: qvg prd
qmm: mrn brh jqq mcp cvd
ftk: ttz hrm
fjl: cvn nld thq
xzl: fck rqv klc
rdz: rtb dxp
qzv: zsz tvq xvj
mmr: dmf mrh
tqf: pcv qsq mjf kfj khg
tkj: vsb jbr dvx
ltd: slh tcf
cgx: mtb tdl xnx nkv
dvd: smk sfv fgd
thh: drd nbr
mfm: njl
xvx: zbc ndz nhv mtr
lvz: xdk
tnh: rjp cdb vnv ckd mfl
rjx: mtr gzf klz
dbf: sdt lsf mhq sbf
vtz: tqh
xdk: jbs
cld: sxf lgv vsm jbr
fdc: snx tsf
bpm: qkv dvj ghb
qck: xss
cdg: gqb
kfm: qsc qxs sls dmr
lgt: knq fbt jqd
ggb: jzb
djv: nvd qpj hnp sft
xmh: gcd hck dtg zlg
vng: tbm nfd knv
lmn: pvr
xcl: jfg tjq
hlg: pnb vkh
vpr: xzn
lpp: xcg kxb
dbq: fgh
jsr: cmq nnf bgf rvt htk
btj: pqd vpn tth
rbd: gqz dmf qbl cpn
tdl: prt
vcj: dpl
hxn: lzx lrt mvg zsj fjq
mfx: qnq lhb qpj
pgk: zkd
nkt: qbl lld fdz
tbt: pzb nqn nzt vgm kkx
lbn: nmt lsf
zjg: zbb htf ckp
ksz: sdn
jsh: vtz
ngh: nrm ntp vlh ccg
nfd: jht
mck: nrm cxm srg
hlj: vsg nrm prd
mqg: vtj zbc klz qvg
xcj: hxj qkv rkl rlm
xtq: tgq mvc bvg
jqq: hdd
cms: vms mbn
mhh: vtc vzr rpc
xts: jxn cjk jht tmg
jlq: xbh fpz rgc kpg dxf
hpz: hnn fmm qct
jkv: mnh bfx jrf djp
rql: vbs fxh xkd nns lsg
zqq: jbc pnj
cdb: rgl
zkz: jxg hkb fzf
clv: vnv
bmh: mkh cms
jqs: tsn vqh ddn
nqr: bsp cxt txr spr vtn
vjq: slh pfv zhp rlz
cgf: khg zvr jng
vdd: fck
lxq: vqh
jvc: qpg jbx jmz xrg
pzx: pjq dql jbr
jmz: tbq
dvj: vsm lhs
szh: vdh hts mdh
cqj: ddm prt
djl: sdn bkx
vhs: qzm
sbs: xtq xbj lkd
jqf: kxv krs prj
qgv: mjg zmn tbp
pdm: fmm mds
prv: scg
vbm: zsr ntl xds qsv
lbb: dvf gks
gtg: gsg
bgg: hzg dqf xnl tdr
bbz: hnh ssv jpd mtm
xvm: nsx
tzp: cbv
zkl: ndk xxd qrf vtn
dqf: jsj bvz
ffz: trv dpx
phf: ffl vvn jpd
zhh: bhq skk jnh mnh
qrf: hss tgm
vbq: vtj
kkt: grn xlk ktt fpd qvg
gmf: fnx tmg dkc nvd
xmt: mdf txg llf vvn
pqh: zcn fhv hcc sfv
psf: xdt rnv
fcm: mhk hck mlq mxp sfn
bpr: ghz bsx tzp knv
hmq: fzm jzk
fff: gzq trh xcf pxd grn
snj: qrf xkq
mnc: dnm
hzs: nmz qln
msr: tlc bmh nhr rcf khg
nxm: lmn
cjg: vcp
ctt: rqv
bcm: btj lhb
jmm: dxf fcf
bff: sml
dxp: cks
tzx: njm spg
dhv: fbt xjk cks
ntr: lsj nbc zsz
bdp: cfh scg
vlz: rbg
tnj: tgv qsg dkd xlb ddd zkd kpg
zlc: mfp lsl dnp bml
knl: kst zbj vkz
fkr: szr tbs sbs
ncd: mtk nfl vbr pdr
vdp: jxv zxv xcg mhc
jmj: xnm nmt hlx dvd
vtn: kfj
gqm: zdd jrs qkp
zdg: nbc vlz
mxl: psb snr qsd
sps: qbz
dkg: mrp nvf xhf bkd
mqj: mnb sng ncr
pdx: vgm lpl jfr
xqc: zdd
klg: nmz hpt pbm phq qzh dfd
rnx: blr sjz prx
ghb: pqc
rzl: gmg dhv
rrg: zdg jhf jsj bsx
rhj: bmd szr kln
czd: snx
jng: gpj jqp
snc: mcp fsd hnk mnc
rtf: mlq ffl
vbb: bpd kcl hbk zfm
hnk: kfs qpc
vqh: xzn
pnc: vnl krp thq
cxd: nph
xxk: tcb bxk sdj
vhl: lgv fkp nrk nsv cjr
hgc: psz ztt
zck: nrt plp
xhm: fvc qsq
fpp: prd jhf hbx
fhv: bxq tmh
dnj: zzq qjl mck kpg
qzt: sbv zbc snr rtn
hlm: tjq
fxh: cvd pjp
rlz: kml cxr gmf
hfq: xzn gjp
npf: jcv vkz
zvk: frt dhh fhs qrh
kdj: qxs
lnp: bgr kzv qhj mqt mpx
vsg: vkz
djk: mnc fpl fhl mkh ttn
vnf: nvg fcr lhs
xbr: zsj
nkf: snq rqh hgs
fnm: bxk zxv qgs
dnr: drm dkd ncc pdt
qct: qln
hmx: dnm
gxj: sxk cgq jsh rzt gkq
hck: kbl csh rzn
hdz: vms fvc zzd
gks: jsm qkv qhz
vcq: lkq sml vqm
sxs: lbn dcd nrd gtx
kdn: zfm dps fqd
kpl: tkq hvq sls ddd vdd
qcp: nbm fcr hdd qfq
ksj: xqc tbc xlv cvq
prd: zcc
fmn: kdl hbx dtv pxc lzd
bjk: cvn vdd qmg xvm jvl zmp tgc jld
dcd: htk
jsm: jfp ggd jbs
lsl: kjt xrj
mnh: cmv
vlt: pjl jzn xms fll cks
mkd: pdd pgk
```

### Challenge year

2023

**Comment by victoraugustofd** [comment]
### Challenge link
https://adventofcode.com/2023/day/25
